README.md

```diff
@@ -79,11 +79,17 @@ Clone this repository to your desired folder: (https://github.com/HERMON-1995/Le
   git clone [https://github.com/HERMON-1995/Leaderboard-list-app-project.git]
```

Comment:
## Your Built With Section
![image](https://user-images.githubusercontent.com/93039359/223364181-f3e4e889-9038-40ba-b492-de059c13061f.png)

## Microverse Readme Template
![image](https://user-images.githubusercontent.com/93039359/223364978-c44924d1-bbda-483c-91f4-086c449a139f.png)
![image](https://user-images.githubusercontent.com/93039359/223364622-177d355b-d41e-40b4-90cd-f7bc28161a2d.png)

**[Optional]**

- [ ] It would be nice if you make your **built with** section as shown in the second image above to make your readme file descriptive and professional :blush:
- [ ] Instead of React.js kindly put HTML, CSS, JavaScript :+1: